docs/sftt/sftt-api-spec.md

```diff
@@ -1681,3 +1681,92 @@ If the site id specified is invalid.
 ##### `401 UNAUTHORIZED`
 
 If the calling user is not an admin.
+
+### Record Stewardship Activity
+
+`POST api/v1/protected/sites/:site_id/record_stewardship`
+
+Records a stewardship activity for the given site. Indicate `True` if the activity (watered, mulched, etc.) was completed, else `False`.
+
+#### Request Body
+
+```json
+{
+  "date": TIMESTAMP,
```

Comment:
Should this be a TIMESTAMP or a date? I'm assuming this field is something the user enters, will they also be adding a time when they performed it?

Reply:
Date might be sufficient and more convenient - I believe NY doesn't ask for time

Reply:
Good point, changed to DATE